src/Misc/tools.ts

```diff
@@ -725,7 +727,7 @@ export class Tools {
         }
         canvas.toBlob(function (blob) {
             successCallback(blob);
-        }, mimeType);
+        }, mimeType, quality);

```

Comment:
is undefined all good here, or should there be a default value ? I guess it should but not sure what their implementation does and I am afraid it would be like using 0 in this case ?

Reply:
[MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob) state `quality` is optional. So I think `canvas.toBlob(cb, type, undefined)` is equivalent to `canvas.toBlob(cb, type)`, right?

I wanted to preserve the defaults the browser chooses, instead of applying our own explicit defaults. And the default is *not* 0, at least when considering the MDN docs linked above!

Reply:
Yup exactly my thoughts, I am just always scared of an internal implementation which relies on arguments.length vs grabbing default when undefined. Keep it like this it should be all good.